build/.vsts-ci.yml

```diff
@@ -17,4 +19,71 @@ jobs:
   steps:
   - template: build.yml
     parameters:
-      packageArtifacts: false
\ No newline at end of file
+      packageArtifacts: false
+      
+- job: Docker
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - task: DockerCompose@0
+    displayName: Build FHIR R4 Server Image
+    inputs:
+      action: Build services
+      azureSubscriptionEndpoint: $(azureSubscriptionEndpoint)
+      azureContainerRegistry: $(azureContainerRegistry)
+      dockerComposeFile: build/docker/docker-compose.yaml
+      dockerComposeFileArgs: FHIR_VERSION = R4
+      projectName: r4
+      additionalImageTags: $(SourceBranchName) 
+
+  - task: DockerCompose@0
+    displayName: Push FHIR R4 Server Image
+    inputs:
+      action: Push services
+      azureSubscriptionEndpoint: $(azureSubscriptionEndpoint)
+      azureContainerRegistry: $(azureContainerRegistry)
+      dockerComposeFile: build/docker/docker-compose.yaml
+      projectName: r4
+      additionalImageTags: $(SourceBranchName) 
```

Comment:
Looks like this could be a template step

Reply:
Refactored into two templates